.pre-commit-config.yaml

```
@@ -17,6 +17,15 @@ repos:
     hooks:
       - id: flake8
         name: Check PEP8
+  
+  - repo: local
+    hooks:
+      - id: mypy
+        name: Check mypy
+        entry: "./scripts/run-mypy"
```

Is it necessary to create a script? Can we just do:
```suggestion
        entry: mypy --namespace-packages --install-types --non-interactive -p camel -p test -p apps
```

---

Truely make sense.